ansible/roles/xq-api/tasks/main.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: Install packages
+  xbps:
+    pkg:
+      - xq-api
+      - snooze
+    state: present
+
+- name: Configure xq-api
+  template:
+    src: xq-api.conf.j2
+    dest: /etc/sv/xq-api/conf
+    owner: root
+    group: root
+    mode: 0644
+  notify:
+    - xq-api
+
+- name: Install xq-api HUP service (1/2)
+  file:
+    path: /etc/sv/xq-api-hup
+    state: directory
+    owner: root
+    group: root
+    mode: 0755
+
+- name: Install xq-api HUP service (2/2)
+  template:
+    src: xq-api-hup.sh.j2
+    dest: /etc/sv/xq-api-hup/run
+    owner: root
+    group: root
+    mode: 0755
+  notify:
+    - xq-api-hup
+
+- name: Create xq-api log directory
+  file:
+    path: /var/log/xq-api
+    state: directory
+    owner: _xqapi
+    group: _xqapi
+    mode: 0755
+
+- name: Enable services
+  runit:
+    name: "{{ item }}"
+    enabled: yes
+  loop:
+    - xq-api
+    - xq-api-hup
+
+- name: Configure nginx base-site
+  include_role:
+    name: nginx
+    tasks_from: base-site
+  vars:
+    - site:
+        name: xq-api
+        urls:
+          - xq-api.voidlinux.org
+        use_ssl: no
```

Comment:
Please use SSL here.

Reply:
Using SSL now -- need you to eyeball that to make sure it's set up correctly, but it seemed pretty clear from other uses.